src/sentry/release_health/duplex.py

```diff
@@ -755,6 +755,18 @@ def run_sessions_query(
         relative_hours = math.ceil((query.end - datetime.now(timezone.utc)).total_seconds() / 3600)
         set_tag("run_sessions_query.rel_end", f"{relative_hours}h")
 
+        project_ids = query.filter_keys.get("project_id")
+        if project_ids and len(project_ids) == 1:
+            project_id = project_ids[0]
+            set_tag("run_sessions_query.project_id", str(project_id))
+            try:
+                project = Project.objects.get_from_cache(id=project_id)
+                assert org_id == project.organization_id
+            except (Project.DoesNotExist, AssertionError):
+                pass
+            else:
+                set_tag("run_sessions_query.platform", project.platform)
```

Comment:
Why this `else` block? I think it's easier to read if you set the tag after the assertion in the `try`.

Reply:
From the Python docs:
> The use of the else clause is better than adding additional code to the [try](https://docs.python.org/3/reference/compound_stmts.html#try) clause because it avoids accidentally catching an exception that wasn’t raised by the code being protected by the try … except statement.

https://docs.python.org/3/tutorial/errors.html#handling-exceptions